packages/forms/package.json

```diff
@@ -42,6 +42,7 @@
     "@talend/assets-api": "^1.2.1",
     "@talend/json-schema-form-core": "1.1.1",
     "@talend/react-components": "^9.2.0",
+    "@talend/design-system": "^7.1.6",
```

Comment:
Are we sure we want to force host app to depend on multiple DS's version ?

Reply:
isn't already what we're doing?

Reply:
yes, that i was checking 😅 
so if no problem with that...